AdvancedTask/AdvancedTask/Pages/Components/ProfileOverview/NotificationComponent.cs

```diff
@@ -0,0 +1,280 @@
+{
+    public class NotificationComponent:BaseClass
+    {
+        private IWebElement SeeAllLink;
+        private IWebElement LoadMoreLink;
+        private IWebElement SeeLessLink;
+        private IWebElement SelectAllLink;
+        private IWebElement UnselectAllLink;
+        private IWebElement MarkAsReadLink;
+        private IWebElement PopUpMessage;
+        private IWebElement SelectNotification;
+        private IWebElement DeleteNotificationLink;
+        
+        public void renderNotificationComponents()
+        {
+            try
+            {
+                SeeAllLink = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/div[1]/div[2]/div/div/div/span/div/div[2]/div/center"));
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+        }
+        public void renderLoadMoreCompnents()
+        {
+            try
+            {
+                LoadMoreLink = driver.FindElement(By.XPath("//a[contains(text(),'Load More...')]"));
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+        }
+        public void renderSeeLessComponents()
+        {
+            try
+            {
+
+                SeeLessLink = driver.FindElement(By.XPath("//a[contains(text(),'...Show Less')]"));
+               
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+        }
+        public void renderSelectAllComponents()
+        {
+            try
+            {
+                SelectAllLink = driver.FindElement(By.XPath("//*[@id=\"notification-section\"]/div[2]/div/div/div[3]/div[1]/div[1]/i"));
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+
+        }
+        public void renderUnselectAllComponents()
+        {
+            try
+            {
+                UnselectAllLink = driver.FindElement(By.XPath("//*[@id=\"notification-section\"]/div[2]/div/div/div[3]/div[1]/div[2]"));
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+
+        }
+        public void renderMarkAsReadComponents()
+        {
+            try
+            {
+                MarkAsReadLink = driver.FindElement(By.XPath("//*[@id=\"notification-section\"]/div[2]/div/div/div[3]/div[1]/div[4]"));
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+
+        }
+        public void renderSelectNotificationsComponents()
+        {
+            try
+            {
+                SelectNotification = driver.FindElement(By.XPath("//*[@id=\"notification-section\"]/div[2]/div/div/div[3]/div[2]/span/span/div/div[1]/div/div/div[3]/input"));
+                                                                 
+            }
```

Comment:
for future reference try to avoid putting try catches every as we should only use them to catch known causes at the moment you are catching Exception but you don't know why and what exception